boards/arm/stm32/stm32f4discovery/src/stm32_bringup.c

```diff
@@ -452,6 +456,16 @@ int stm32_bringup(void)
     }
 #endif
 
+#ifdef CONFIG_INPUT_SBUTTON
+  /* Register the Single Button Dual Action driver */
+
+  ret = board_sbutton_initialize(0);
```

Comment:
I would expect function with such name to have external linkage and be called outside of specific board. Shouldn't that be prefixed with something like `stm32_` as you have in comment for the function? This seems internal to stm32f4disco.